src/sass/atoms/_flex.scss

```diff
@@ -0,0 +1,57 @@
+$flexClasses: (
+    flex: ( display: flex ),
+
+    flex-ai-fs: ( align-items: flex-start ),
+    flex-ai-fe: ( align-items: flex-end ),
+    flex-ai-c: ( align-items: center ),
+    flex-ai-b: ( align-items: baseline ),
+    flex-ai-s: ( align-items: stretch ),
+    flex-ai-inherit: ( align-items: inherit ),
+    flex-ai-initial: ( align-items: initial ),
+    
+    flex-jc-fs: ( justify-content: flex-start ),
+    flex-jc-fe: ( justify-content: flex-end ),
+    flex-jc-c: ( justify-content: center ),
+    flex-jc-sa: ( justify-content: space-around ),
+    flex-jc-sb: ( justify-content: space-between ),
+    flex-jc-se: ( justify-content: space-evenly ),
+    flex-jc-initial: ( justify-content: initial ),
+    flex-jc-inherit: ( justify-content: inherit ),
+
+    flex-wrap-0: ( flex-wrap: nowrap ),
+    flex-wrap-w: ( flex-wrap: wrap ),
+    flex-wrap-wr: ( flex-wrap: wrap-reverse ),
+    flex-wrap-inherit: ( flex-wrap: inherit ),
+
+    flex-d-c: ( flex-direction: column ),
+    flex-d-r: ( flex-direction: row ),
```

Comment:
How about just flex-column, flex-row? Breaks backwards compatibility, but that's likely to happen anyway for those classes.